gapil/compiler/plugins/encoder/test/BUILD.bazel

```diff
@@ -51,7 +51,7 @@ go_library(
         "//gapil/runtime/cc",
     ],
     cgo = True,
-    copts = cc_copts(),  # keep
+    cppopts = cc_copts(),
```

Comment:
Why is this rule (out of all the hundreds of others) special?

Reply:
This appears to be the only place where we are asking cgo to compile C++ code, compared to just C code. `rules_go` added a feature where you now can specify flags for them separately (`copts` and `cxxopts`) and `cppopts` which will be used by both. We could really change all instances of `copts` to `cppopts`.